sources/packages/web/src/components/layouts/aest/AESTHomeSideBar.vue

```diff
@@ -1,51 +1,68 @@
 <template>
-  <v-navigation-drawer app class="body-background" permanent>
-    <v-list density="compact" nav>
+  <v-navigation-drawer app color="background" permanent>
+    <v-list
+      active-class="active-sidebar-item"
+      density="compact"
+      bg-color="background"
+      active-color="primary"
+    >
       <v-list-item
         v-for="topItem in topItems"
         :key="topItem.label"
         :prepend-icon="topItem.icon"
         :title="topItem.label"
-        @click="topItem.command"
+        :to="topItem.command()"
       />
     </v-list>
-    <v-list density="compact" nav>
-      <v-list-subheader>Student requests</v-list-subheader>
+    <v-list
+      active-class="active-sidebar-item"
+      density="compact"
+      bg-color="background"
+      active-color="primary"
+    >
+      <v-list-subheader class="nav-subtitle">Student requests</v-list-subheader>
       <v-list-item
         :prepend-icon="studentAccountApplicationItem.icon"
         :title="studentAccountApplicationItem.label"
-        @click="studentAccountApplicationItem.command"
+        :to="studentAccountApplicationItem.command()"
       />
       <v-list-item
         :prepend-icon="exceptionsItem.icon"
         :title="exceptionsItem.label"
-        @click="exceptionsItem.command"
+        :to="exceptionsItem.command()"
       />
       <v-list-item
         :prepend-icon="appealsItem.icon"
         :title="appealsItem.label"
-        @click="appealsItem.command"
+        :to="appealsItem.command()"
       />
     </v-list>
-    <v-list density="compact" nav>
-      <v-list-subheader>Institution requests</v-list-subheader>
+    <v-list
+      active-class="active-sidebar-item"
+      density="compact"
+      bg-color="background"
+      active-color="primary"
+    >
+      <v-list-subheader class="nav-subtitle"
+        >Institution requests</v-list-subheader
+      >
       <v-list-item
         :title="designations.label"
         :prepend-icon="designations.icon"
-        @click="designations.command"
+        :to="designations.command()"
       />
       <v-list-item
         :title="offerings.label"
         :prepend-icon="offerings.icon"
-        @click="offerings.command"
+        :to="offerings.command()"
       />
     </v-list>
     <template #append>
       <v-list density="compact" nav>
         <check-permission-role :role="Role.AESTReports">
           <template #="{ notAllowed }">
             <v-list-item
-              @click="reports.command"
+              :to="reports.command()"
```

Comment:
I believe that there is no reason to call a method here. The value can be directly set in the same way as the other ones.